docs/EXTERNAL_CHECK_CREATION.md

```diff
@@ -41,6 +40,13 @@ Simply build your program into a container, `docker push` it to somewhere your c
 
 Clients outside of Go can be found in the [clients directory](../clients).
 
+#### Injected Check Pod Environment Variables
+
```

Comment:
```suggestion
The following environment variables are injected into every checker pod that Kuberhealthy runs.  When writing your checker code, you can depend on these environment variables always being available to you, even if you do not specify them in your `khcheck` spec..
```

Did we get all of them?  Can you check that environment variable injecting code you deduplicated a while ago and verify please?

Reply:
These are not all of them -- but these are the ones that checks should be aware of.

The other ones are `UUID`, `external check POST URL`, etc.